docs/_posts/examples/.eslintrc

```diff
@@ -5,7 +5,8 @@
     "MapboxGeocoder": true,
     "MapboxDirections": true,
     "turf": true,
-    "d3": true
+    "d3": true,
+    "Uint8Array": true
```

Comment:
Do you remember why we have `es6` env set to `false` a few lines below? As far as I remember, turning this back to `true` would allow not specifying common globals like this.